src/Uno.UI.Tests/App/Views/StylesTestControl.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Uno.UI.Tests.App.Views
+{
+	public partial class StylesTestControl : Control
+	{
+		public string Rugosity
+		{
+			get { return (string)GetValue(RugosityProperty); }
+			set { SetValue(RugosityProperty, value); }
+		}
+
+		// Using a DependencyProperty as the backing store for Rugosity.  This enables animation, styling, binding, etc...
+		public static readonly DependencyProperty RugosityProperty =
+			DependencyProperty.Register("Rugosity", typeof(string), typeof(StylesTestControl), new PropertyMetadata("FromDefault"));
+	}
+
+	public partial class StylesTestButton : Button
+	{
+		public object ExposedKey => DefaultStyleKey;
+
+		public string Animosity
+		{
+			get { return (string)GetValue(AnimosityProperty); }
+			set { SetValue(AnimosityProperty, value); }
+		}
+
+		// Using a DependencyProperty as the backing store for Animosity.  This enables animation, styling, binding, etc...
+		public static readonly DependencyProperty AnimosityProperty =
+			DependencyProperty.Register("Animosity", typeof(string), typeof(StylesTestButton), new PropertyMetadata("FromDefault"));
+
+
+	}
+
+	public partial class StylesTestButtonCustomKey : Button
+	{
+		public StylesTestButtonCustomKey()
+		{
+			DefaultStyleKey = typeof(StylesTestButtonCustomKey);
+		}
+	}
+
+	public partial class StylesTestRadioButton : RadioButton
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['partial' is gratuitous in this context.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826539)

Reply:
I can see how you might get that impression robo-Carl, but... no.